test/e2e/sgx/sgx.go

```diff
@@ -112,5 +114,39 @@ func describe() {
 
 		ginkgo.By("waiting the pod to finish successfully")
 		e2epod.NewPodClient(f).WaitForSuccess(pod.ObjectMeta.Name, 60*time.Second)
+
+		e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "delete", "-k", filepath.Dir(deploymentPluginPath))
```

Comment:
If I understood how e2e works correctly from what I have done...
After each It(), namespace gets deleted, and actually the plugin should be also deleted automatically after the first It() run.
Is this necessary?

Reply:
it was added after my previous comment (see above) although it indeed did not change the part that I thought was a false positive due to an earlier device plugin instance running

Reply:
@hj-johannes-lee 
It() implicitly deletes the test's namespace. There was a confusion with interpreting the logs which was tested with the isolated run of operator SGX.